### PR TITLE
Added missing translation strings

### PR DESCRIPTION
**Pull request by Caleydon**

### CODE REVIEW

File: admin-dev/themes/default/template/controllers/translations/helpers/view/translation_errors.tpl [hunk 1]
```diff
@@ -69,9 +69,9 @@
 					{l s='You MUST use this syntax in your translations. Here are a few examples:'}
 				</p>
 				<ul>
-					<li>"There are <strong>%d</strong> products": {l s='"%s" will be replaced by a number.' sprintf='%d'}</li>
-					<li>"List of pages in <strong>%s</strong>": {l s='"%s" will be replaced by a string.' sprintf='%s'}</li>
-					<li>"Feature: <strong>%1$s</strong> (<strong>%2$d</strong> values)": {l s='The numbers enable you to reorder the variables when necessary.'}</li>
+					<li>"{l s='There are'} <strong>%d</strong> {l s='products'}": {l s='"%s" will be replaced by a number.' sprintf='%d'}</li>
```
Comment (jeromenadaud) [inline admin-dev/themes/default/template/controllers/translations/helpers/view/translation_errors.tpl]: Hi, 
Thank you for your PR, maybe you have not noticed that you can use {l s='There are [1]%d[/1] products' tags=['<strong>']}

Best regards.

**Comment by Caleydon** [comment]
Sure, but this is an example of using variables in the top of translation pages.

**Comment by jeromenadaud** [comment]
Yes you are right, so maybe {l s='There are %s products' sprintf='<strong>%d</strong>'} like this we can preserve RTL. what do you think ?

Best regards.

**Comment by jeromenadaud** [comment]
Hi,

Any feedback ?

Best regards

**Comment by Caleydon** [comment]
Please fix it if you have better solution. At last, it's PrestaShop bug. Thanks.

**Comment by jeromenadaud** [comment]
Hi,

Thank you for your PR.

Best regards.

**Pull request merged**